### DISCUSSION TITLE
0.1.2 Alpha

### DISCUSSION THREAD

**Discussion by philipbaileynar** [Riverscapes/QRiS#105] [Announcements]
@joewheaton here is an extremely alpha release of QRiS. Please review and create issues in as orderly manner as possible.

- Assign layer and symbology tickets to @nick4rivers 
- Assign all other issues to me.

Caveats

- Please do not share with anyone else yet. You are going to have significant feedback and the fewer hands that this is in the better.
- Strong possibility that feedback will require restructuring the QRiS project GeoPackage. If this occurs during this alpha stage then future versions might not be compatible with projects that you create with this version. (Once we go to production then we will support upgrading users projects to ensure forward compatibility for data).

<hr /><em>This discussion was created from the release <a href='https://github.com/Riverscapes/QRiS/releases/tag/0.1.2-alpha'>0.1.2 Alpha</a>.</em>

**Comment by joewheaton** [comment]
On it @philipbaileynar.
